pkg/nsx/services/securitypolicy/firewall.go

```diff
@@ -41,14 +41,21 @@ func InitializeSecurityPolicy(service common.Service) (*SecurityPolicyService, e
 	fatalErrors := make(chan error)
 
 	wg.Add(3)
+
 	securityPolicyService := &SecurityPolicyService{Service: service}
-	securityPolicyService.GroupStore = cache.NewIndexer(common.KeyFunc, cache.Indexers{util.TagScopeSecurityPolicyCRUID: common.IndexFunc(util.TagScopeSecurityPolicyCRUID)})
-	securityPolicyService.SecurityPolicyStore = cache.NewIndexer(common.KeyFunc, cache.Indexers{util.TagScopeSecurityPolicyCRUID: common.IndexFunc(util.TagScopeSecurityPolicyCRUID)})
-	securityPolicyService.RuleStore = cache.NewIndexer(common.KeyFunc, cache.Indexers{util.TagScopeSecurityPolicyCRUID: common.IndexFunc(util.TagScopeSecurityPolicyCRUID)})
+	operateSecurityPolicyStore = &OperateSecurityPolicyStore{securityPolicyService}
```

Comment:
If operateSecurityPolicyStore embedded a Indexer, we could call operateSecurityPolicyStore.GetByKey directly instead of need both operateSecurityPolicyStore/securityPolicyStore

Reply:
ok